<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>960</width>
    <height>594</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>960</width>
    <height>500</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <property name="styleSheet">
   <string notr="true">background-color: rgb(255, 255, 255);
background-color: rgb(0, 100, 200);</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <layout class="QGridLayout" name="gridLayout">
    <item row="0" column="0">
     <layout class="QHBoxLayout" name="horizontalLayout">
      <item>
       <layout class="QVBoxLayout" name="verticalLayout_2">
        <item>
         <widget class="QLabel" name="figureLabel">
          <property name="minimumSize">
           <size>
            <width>320</width>
            <height>240</height>
           </size>
          </property>
          <property name="maximumSize">
           <size>
            <width>320</width>
            <height>240</height>
           </size>
          </property>
          <property name="styleSheet">
           <string notr="true">background-color: rgb(255, 0, 255);</string>
          </property>
          <property name="text">
           <string/>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLabel" name="objectFindLabel">
          <property name="minimumSize">
           <size>
            <width>320</width>
            <height>240</height>
           </size>
          </property>
          <property name="maximumSize">
           <size>
            <width>320</width>
            <height>240</height>
           </size>
          </property>
          <property name="font">
           <font>
            <family>Courier New</family>
            <pointsize>13</pointsize>
           </font>
          </property>
          <property name="styleSheet">
           <string notr="true">background-color: rgb(255, 255, 0);</string>
          </property>
          <property name="text">
           <string/>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item>
       <widget class="QLabel" name="imageLabel">
        <property name="minimumSize">
         <size>
          <width>640</width>
          <height>480</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>640</width>
          <height>480</height>
         </size>
        </property>
        <property name="styleSheet">
         <string notr="true">background-color: rgb(0, 0, 0);</string>
        </property>
        <property name="text">
         <string>TextLabel</string>
        </property>
       </widget>
      </item>
     </layout>
    </item>
    <item row="1" column="0">
     <layout class="QHBoxLayout" name="horizontalLayout_2">
      <item>
       <widget class="QPushButton" name="startButton">
        <property name="minimumSize">
         <size>
          <width>100</width>
          <height>80</height>
         </size>
        </property>
        <property name="palette">
         <palette>
          <active>
           <colorrole role="Button">
            <brush brushstyle="SolidPattern">
             <color alpha="255">
              <red>102</red>
              <green>255</green>
              <blue>0</blue>
             </color>
            </brush>
           </colorrole>
           <colorrole role="Base">
            <brush brushstyle="SolidPattern">
             <color alpha="255">
              <red>102</red>
              <green>255</green>
              <blue>0</blue>
             </color>
            </brush>
           </colorrole>
           <colorrole role="Window">
            <brush brushstyle="SolidPattern">
             <color alpha="255">
              <red>102</red>
              <green>255</green>
              <blue>0</blue>
             </color>
            </brush>
           </colorrole>
          </active>
          <inactive>
           <colorrole role="Button">
            <brush brushstyle="SolidPattern">
             <color alpha="255">
              <red>102</red>
              <green>255</green>
              <blue>0</blue>
             </color>
            </brush>
           </colorrole>
           <colorrole role="Base">
            <brush brushstyle="SolidPattern">
             <color alpha="255">
              <red>102</red>
              <green>255</green>
              <blue>0</blue>
             </color>
            </brush>
           </colorrole>
           <colorrole role="Window">
            <brush brushstyle="SolidPattern">
             <color alpha="255">
              <red>102</red>
              <green>255</green>
              <blue>0</blue>
             </color>
            </brush>
           </colorrole>
          </inactive>
          <disabled>
           <colorrole role="Button">
            <brush brushstyle="SolidPattern">
             <color alpha="255">
              <red>102</red>
              <green>255</green>
              <blue>0</blue>
             </color>
            </brush>
           </colorrole>
           <colorrole role="Base">
            <brush brushstyle="SolidPattern">
             <color alpha="255">
              <red>102</red>
              <green>255</green>
              <blue>0</blue>
             </color>
            </brush>
           </colorrole>
           <colorrole role="Window">
            <brush brushstyle="SolidPattern">
             <color alpha="255">
              <red>102</red>
              <green>255</green>
              <blue>0</blue>
             </color>
            </brush>
           </colorrole>
          </disabled>
         </palette>
        </property>
        <property name="font">
         <font>
          <family>Adobe Caslon Pro</family>
          <pointsize>24</pointsize>
          <weight>50</weight>
          <italic>true</italic>
          <bold>false</bold>
         </font>
        </property>
        <property name="styleSheet">
         <string notr="true">background-color: rgb(102, 255, 0);</string>
        </property>
        <property name="text">
         <string>start</string>
        </property>
        <property name="checkable">
         <bool>false</bool>
        </property>
        <property name="default">
         <bool>false</bool>
        </property>
        <property name="flat">
         <bool>true</bool>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="stopButton">
        <property name="minimumSize">
         <size>
          <width>100</width>
          <height>80</height>
         </size>
        </property>
        <property name="font">
         <font>
          <family>Adobe Caslon Pro</family>
          <pointsize>24</pointsize>
          <italic>true</italic>
         </font>
        </property>
        <property name="styleSheet">
         <string notr="true">background-color: rgb(102, 255, 0);ss</string>
        </property>
        <property name="text">
         <string>stop</string>
        </property>
        <property name="flat">
         <bool>true</bool>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="addFaceButton">
        <property name="minimumSize">
         <size>
          <width>100</width>
          <height>80</height>
         </size>
        </property>
        <property name="font">
         <font>
          <family>Adobe Caslon Pro</family>
          <pointsize>24</pointsize>
          <italic>true</italic>
         </font>
        </property>
        <property name="styleSheet">
         <string notr="true">background-color: rgb(102, 255, 0);</string>
        </property>
        <property name="text">
         <string>add Face</string>
        </property>
        <property name="flat">
         <bool>true</bool>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="saveModelButton">
        <property name="minimumSize">
         <size>
          <width>100</width>
          <height>80</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>1677215</width>
          <height>1677215</height>
         </size>
        </property>
        <property name="font">
         <font>
          <family>Adobe Caslon Pro</family>
          <pointsize>24</pointsize>
          <italic>true</italic>
         </font>
        </property>
        <property name="styleSheet">
         <string notr="true">background-color: rgb(102, 255, 0);</string>
        </property>
        <property name="text">
         <string>save Data</string>
        </property>
        <property name="flat">
         <bool>true</bool>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="pushButton_4">
        <property name="minimumSize">
         <size>
          <width>100</width>
          <height>80</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>1677215</width>
          <height>1677215</height>
         </size>
        </property>
        <property name="font">
         <font>
          <family>Adobe Caslon Pro</family>
          <pointsize>24</pointsize>
          <italic>true</italic>
         </font>
        </property>
        <property name="styleSheet">
         <string notr="true">background-color: rgb(102, 255, 0);</string>
        </property>
        <property name="text">
         <string>recognition</string>
        </property>
        <property name="flat">
         <bool>true</bool>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="findBallButton">
        <property name="minimumSize">
         <size>
          <width>100</width>
          <height>80</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>1677215</width>
          <height>1677215</height>
         </size>
        </property>
        <property name="font">
         <font>
          <family>Adobe Caslon Pro</family>
          <pointsize>24</pointsize>
          <italic>true</italic>
         </font>
        </property>
        <property name="styleSheet">
         <string notr="true">background-color: rgb(102, 255, 0);</string>
        </property>
        <property name="text">
         <string>track Ball</string>
        </property>
        <property name="flat">
         <bool>true</bool>
        </property>
       </widget>
      </item>
     </layout>
    </item>
   </layout>
   <widget class="QPushButton" name="startButton_2">
    <property name="geometry">
     <rect>
      <x>420</x>
      <y>420</y>
      <width>100</width>
      <height>80</height>
     </rect>
    </property>
    <property name="minimumSize">
     <size>
      <width>100</width>
      <height>80</height>
     </size>
    </property>
    <property name="font">
     <font>
      <family>Adobe Caslon Pro</family>
      <pointsize>24</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(102, 255, 0);</string>
    </property>
    <property name="text">
     <string>start</string>
    </property>
    <property name="default">
     <bool>false</bool>
    </property>
    <property name="flat">
     <bool>true</bool>
    </property>
   </widget>
  </widget>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
